examples/jokes/README.md

```diff
@@ -22,10 +22,12 @@ This example demonstrates some of the basic features of Remix, including:
 - Resource Routes
 - Deployment
 
-This is the finished version of [the tutorial](https://remix.run/tutorials/jokes)
+This is the finished version of [the tutorial](https://remix.run/tutorials/jokes).
 
 - [Remix Docs](https://remix.run/docs)
 
+> For comparison, a variant of this sample app using EdgeDB instead of SQLite/Prisma is [available here](https://github.com/edgedb/edgedb-examples/tree/main/remix) ([see the Git diff](https://github.com/edgedb/edgedb-examples/compare/d654bcc1...ad86e66b)).
```

Comment:
Would you rather make this compare link compare branches instead? That way if you realize there's an easier/better way to do it you can update the branch without having to update this link.

If not that's fine too. Happy to merge as-is!

Reply:
I wasn't sure how to do that exactly, because the latest version of the `remix` app will always be in the `main` branch. But if I do a comparison from a `jokes-initial...main`, there will be a bunch of changes mixed in relating to other non-Remix examples (because our `edgedb/edgedb-examples` repo holds sample projects for a variety of frameworks. Am I missing something obvious here?

Reply:
Ah, I didn't look carefully enough and didn't notice the fact that this is in your examples repo. I thought you were going to make a completely separate repo.

I think you could accomplish the same with tags. But again, it's not a big deal. Let me know if you'd rather I just merge it as-is :)

Reply:
Ok I decided we should get rid of the link entirely for the time being 😅 

- It's probably a bit too much to include in a random aside in this README
- I'm worried about that diff going stale as Remix and EdgeDB change their APIs/workflows
- I don't want to bother you guys with PRs down the road just to update this darn link lol

I'll include the link on our end instead so people who want to dig deeper can explore it. Anyway - ready to merge. Thanks Kent :)